common/src/cellular_pktio.c

```diff
@@ -38,7 +38,9 @@
 #include "cellular_types.h"
 #include "cellular_internal.h"
 #include "cellular_pktio_internal.h"
-#include "cellular_common_internal.h"
+#ifdef TEST
```

Comment:
Why only in TEST?

Reply:
Will remove this one. seems it's unnecessary.